amp_filter.rb

```diff
@@ -21,7 +21,7 @@ def amp_images(input, responsive = true, wi = nil, he = nil)
           else
             # FastImage doesn't seem to handle local paths when used with Jekyll
             # so let's just force the path
-            src = File.join(Dir.pwd, image['src'])
+            src = File.join(Dir.pwd + '_site', image['src'])
```

Comment:
Should the `_site` part in the path also be a separate parameter to `File.join`? In my local environment `pwd` didn't add the slash to the end and the path got malformed.

I would change `File.join(Dir.pwd + '_site', image['src'])` to `File.join(Dir.pwd, '_site', image['src'])`.